src/Services/Person/Person.Test/FetchPersonDetails/ComponentTests/FetchPersonDetailsTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Net;
+using System.Text.Json;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Moq.Protected;
+using Person.Application.FetchPersonDetails.Repositories;
+using Person.Infrastructure.TmdbDtos.PersonDetailsDto;
+using Person.Infrastructure.Util.Mappers;
+using Person.Test.FetchPersonDetails.Comparers;
+using Person.Test.Shared;
+
+namespace Person.Test.FetchPersonDetails.ComponentTests;
+
+[TestFixture]
+public class FetchPersonDetailsTest
+{
+    private Mock<ILogger<FetchPersonDetailsRepository>> _loggerMock;
+    private IFetchPersonDetailsRepository _fetchPersonDetailsRepository;
+    private readonly string _apiApi =
+        Environment.GetEnvironmentVariable("TMDB_API_KEY");
+
+    private PersonDetailsComparer _personDetailsComparer;
+
+    [SetUp]
+    public void Setup()
+    {
+        _loggerMock = new Mock<ILogger<FetchPersonDetailsRepository>>();
+        _personDetailsComparer = new PersonDetailsComparer();
+    }
+
+    [Test]
+    public async Task FetchPersonDetails_WithPersonID2_ShouldReturnCorrectData()
+    {
+        // Arrange
+        const string file =
+            "FetchPersonDetails/ComponentTests/Fakes/PersonDetailsWithId2Response.json";
```

Comment:
Not sure how it works after compiling the project, but this path might break when running in CI environment 🔥

Reply:
Yeah, it should read the relative path in the test project, so in theory, it should work 🤞